pkg/app/web/src/components/settings-page/piped/components/piped-table-row.tsx

```diff
@@ -142,7 +160,19 @@ export const PipedTableRow: FC<Props> = memo(function PipedTableRow({
         className={clsx({ [classes.disabledItem]: piped.disabled })}
       >
         <TableCell>
-          <Typography variant="subtitle2">{piped.name}</Typography>
+          <Typography variant="subtitle2">
+            {piped.name}
```

Comment:
Let's show the status before the name because the name length is variable.

Reply:
Nice point

Reply:
It will look like this. I just worry about do they missed it with the line header or something 😂  And since the name of the column is `name` so I guess the name should have a higher priority and should be placed first.

<img width="1440" alt="Screen Shot 2021-12-07 at 17 59 08" src="https://user-images.githubusercontent.com/32532742/144998244-2385dddf-44bf-4793-b97d-f576fa582725.png">

Reply:
I don't have a strong opinion about that, but plz let me know you guys' thought 😄

Reply:
For sure, seeing it actually, we can tell it's a little bit confusing.

Reply:
Just started feeling that it's slightly better to put them on the right side.

Reply:
👍

Reply:
🙆‍♀️